astroquery/irsa/core.py

```diff
@@ -131,7 +131,7 @@ def query_region(self, coordinates=None, catalog=None, spatial='Cone',
             details). Required if spatial is ``'Cone'`` or ``'Box'``. Optional
             if spatial is ``'Polygon'``.
         catalog : str
-            The catalog to be used (see the *Notes* section below).
+            The catalog to be used can be found using ``'Irsa.print_catalogs()'`` which returns a dictionary of catalog names with their description.
```

Comment:
Please wrap the lines around 80 chars, and you can also link to the API docs by using only single backticks as `` :meth:`~astroquery.irsa.IrsaClass.print_catalogs( )` `` (see examples below)